src/sentry/models/dashboard_widget.py

```diff
@@ -151,6 +151,7 @@ class OnDemandExtractionState(models.TextChoices):
         ENABLED_MANUAL = "enabled:manual", gettext_lazy("enabled:manual")
         """ This widget query was enabled manually post creation or otherwise. """
 
+    spec_version = models.IntegerField(null=True)
```

Comment:
Do we want a default version here?

Reply:
No, the whole table is filled in with a task checking for cardinality issues every ~1 day so I can do it there, I need to start forking the versions as well.